test/jdk/java/io/File/GetCanonicalPath.java

```diff
@@ -80,21 +81,50 @@ private static Stream<Arguments> pathProvider() {
         return list.stream();
     }
 
+    private static Stream<Arguments> pathProviderUnix() {
+        List<Arguments> list = new ArrayList<Arguments>();
+
+        list.add(Arguments.of("/../../../../../a/b/c",
+                              "/a/b/c"));
+        list.add(Arguments.of("/../../../../../a/../b/c",
+                              "/b/c"));
+        list.add(Arguments.of("/../../../../../a/../../b/c",
+                              "/b/c"));
+        list.add(Arguments.of("/../../../../../a/../../../b/c",
+                              "/b/c"));
+        list.add(Arguments.of("/../../../../../a/../../../../b/c",
+                              "/b/c"));
+
+        return list.stream();
```

Comment:
You could use Stream.of here, e.g.

```
        return Stream.of(
            Arguments.of("/../../../../../a/b/c", "/a/b/c"),
            Arguments.of("/../../../../../a/../b/c", "/b/c"),
            Arguments.of("/../../../../../a/../../b/c", "/b/c"),
            Arguments.of("/../../../../../a/../../../b/c", "/b/c"),
            Arguments.of("/../../../../../a/../../../../b/c", "/b/c")
        );
```

Reply:
So changed in cd2b3776463ddc95b36f7b65be12da7ac2686c53.